recipes/django-pint/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "django-pint" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/django-pint-{{ version }}.tar.gz
+  sha256: dbcbe5143cc8feb83e03ce8e4a517dc370f918bbb96343c677ad84cebe7dc3b7
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - setuptools >=46.1.0
+    - setuptools-scm >=5
+    - wheel
+    - pip
+  run:
+    - python >=3.8
+    - django >=3.2
+    - pint >=0.16
+
+test:
```

Comment:
Please add some import tests:

```
test:
   ...
   imports:
       - django_pint
       - quantityfield
```

Reply:
added these using a run_test.py because I needed to set the DJANGO_SETTINGS_MODULE (or `django.setup()`) for the imports to succeed apparently